README.rst

```diff
@@ -49,6 +49,12 @@ Afterwards, install the add-on via Add-on Controlpanel
 Enable the Field Collapsing behavior on the Collection Content Type by visitng Dexterity Content Types Controlpanel.
 
 
+TODOs
+----------
+- Add support for Live Preview of the field collapsing feature
+- Add multiselect support, whereby more than one field can be collapsed on.
```

Comment:
needed now to prevent issues with upgrades

Reply:
Are you talking about upgrade steps? If so, using the behaviour approach, this will never use an upgrade step or require any upgrades. It's using Select2 widget.

Reply:
you are storing a single value now in the DB and a list in the future. That makes you future code more complex. The schema will change. Please just do the work to make it select multiple metadata fields now. its not that hard.